backend/src/main/java/com/eyetrade/backend/model/entity/User.java

```diff
@@ -69,4 +70,9 @@
 
     @Column(name = "confirmed")
     private boolean confirmed;
+
+    @NotNull
+    @Column(name = "status")
```

Comment:
What is the meaning of status? I couldn't understand

Reply:
I think we can change more specific name such as privacy type

Reply:
agree, it would make more sense